Pipfile

```diff
@@ -0,0 +1,94 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+alembic = "==1.9.4"
+aniso8601 = "==8.0.0"
+asgiref = "==3.6.0"
+attrs = "==22.2.0"
+bcrypt = "==3.1.7"
+bidict = "==0.22.1"
+blinker = "==1.5"
+boto3 = "==1.26.91"
+botocore = "==1.29.91"
+caer = "==2.0.8"
+certifi = "==2022.12.7"
+cffi = "==1.15.1"
+charset-normalizer = "==3.0.1"
+click = "==7.1.2"
+config = "==0.5.1"
+contourpy = "==1.0.7"
+cryptography = "==39.0.2"
+cycler = "==0.11.0"
+datetime = "==5.1"
+depthai = "==2.20.2.0"
+distlib = "==0.3.6"
+exceptiongroup = "==1.1.0"
+filelock = "==3.9.0"
+flask = "==1.1.2"
+flask-bcrypt = "==1.0.1"
+flask-cors = "==3.0.10"
+flask-mail = "==0.9.1"
+flask-migrate = "==4.0.4"
+flask-restful = "==0.3.9"
+flask-socketio = "==5.3.3"
+flask-sqlalchemy = "==2.4.3"
+fonttools = "==4.38.0"
+gevent = "==22.10.2"
+greenlet = "==2.0.2"
+idna = "==3.4"
+imutils = "==0.5.4"
+iniconfig = "==2.0.0"
+itsdangerous = "==1.1.0"
+jinja2 = "==2.11.2"
+jmespath = "==1.0.1"
+joblib = "==1.2.0"
+kiwisolver = "==1.4.4"
+mako = "==1.2.4"
+markupsafe = "==1.1.1"
+mypy = "==0.991"
+mypy-extensions = "==0.4.3"
+packaging = "==23.0"
+pbr = "==5.11.1"
+pillow = "==9.4.0"
+platformdirs = "==2.6.2"
+pluggy = "==1.0.0"
+protobuf = "==4.21.12"
+psycopg2-binary = "==2.9.5"
+pyignite = "==0.6.1"
+pyjwt = "==2.6.0"
+pylogix = "==0.8.7"
+pyparsing = "==3.0.9"
+pytest = "==7.2.2"
+python-dateutil = "==2.8.2"
+python-dotenv = "==1.0.0"
+python-engineio = "==4.4.0"
+python-http-client = "==3.3.7"
+pytz = "==2020.1"
+requests = "==2.28.2"
+sendgrid = "==6.8.0"
+six = "==1.15.0"
+smtpapi = "==0.3.1"
+sqlalchemy = "==1.3.18"
+sqlparse = "==0.4.3"
+starkbank-ecdsa = "==2.2.0"
+stevedore = "==4.1.1"
+threadpoolctl = "==3.1.0"
+tomli = "==2.0.1"
+twilio = "==7.17.0"
+typing-extensions = "==4.4.0"
+urllib3 = "==1.26.14"
+websocket = "==0.2.1"
+werkzeug = "==1.0.1"
+"zope.event" = "==4.6"
+"zope.interface" = "==6.0"
+flask-login = "==0.4.1"
+oauthlib = "==3.0.1"
+pyopenssl = "==19.0.0"
+
+[dev-packages]
+
```

Comment:
Libraries for formatting and linting, as well as testing libraries should be installed as dev dependencies

For now, this is fine. But if you plan to setup CI/CD pipeline then we will need to add more things here